Xamarin.Forms.Platform.iOS/Renderers/EntryRenderer.cs

```diff
@@ -302,5 +310,64 @@ void UpdateReturnType()
 			Control.ReturnKeyType = Element.ReturnType.ToUIReturnKeyType();
 		}
 
+		bool _selectedTextRangeIsUpdating;
+
+		void UpdateCursorFromControl(NSObservedChange obj)
+		{
+			if (_selectedTextRangeIsUpdating)
```

Comment:
Do we also want to do some null check on `Control`/`Element`?

Reply:
Not sure if they can be null here, but I'm not sure...

Reply:
Good question. I could add them just in case.

Reply:
yeah i would add it as well